python/ray/tests/test_runtime_context.py

```diff
@@ -338,6 +338,90 @@ def test_errors_when_ray_not_initialized():
         ray.get_runtime_context().get_node_id()
 
 
+def test_threaded_actor_runtime_context(ray_start_regular):
+    @ray.remote
+    class ThreadedActor:
+        def task_id(self):
+            return ray.get_runtime_context().get_task_id()
+
+    threaded_actor = ThreadedActor.options(max_concurrency=3).remote()
+    # Test threaded actor
+    tasks = [threaded_actor.task_id.remote() for _ in range(10)]
+    submitted_task_ids = [task.task_id().hex() for task in tasks]
+    actual_task_ids = ray.get(tasks)
+    assert ",".join(submitted_task_ids) == ",".join(actual_task_ids)
+
+
+def test_async_actor_runtime_context(ray_start_regular):
+    @ray.remote
+    class AsyncActor:
+        async def task_id(self):
+            return ray.get_runtime_context().get_task_id()
+
+    # Asyncio actors run in another thread that's beyond knowledge of ray
+    async_actor = AsyncActor.options(max_concurrency=1).remote()
+    tasks = [async_actor.task_id.remote() for _ in range(10)]
+    expect_task_ids = [None] * len(tasks)
```

Comment:
So, is this the right behavior we should have for async actor now? 

If `max_concurrency = 1`, we could actually return `current_exec_context` to make it work. But if `max_concurrency > 1`, we have no way to know which task the thread is executing.